owncloudApp/src/main/java/com/owncloud/android/presentation/settings/SettingsActivity.kt

```diff
@@ -70,7 +71,7 @@ class SettingsActivity : AppCompatActivity() {
             is SettingsMoreFragment -> R.string.prefs_subsection_more
             else -> R.string.actionbar_settings
         }
-
+        setTitle(titleId)
```

Comment:
Maybe with this, that changes the title dynamically depending on the section you are at, the label in the Manifest is not needed anymore for this activity

Reply:
Yeah, I tested and it is not needed. 
Also I added a label to `LogsListActivity`, because when we navigate to list of logs, the TalkBack said: owncloud. I also remove the `setTitle(R.string.prefs_log_open_logs_list_view)` in the activity, because is not needed anymore.